lib/phoenix/logger.ex

```diff
@@ -85,6 +88,30 @@ defmodule Phoenix.Logger do
   except those that match exactly `id` or `order`. If a kept parameter
   matches, all parameters nested under that one will also be kept.
 
+  ## Dynamic log level
+
+  In some cases you may wish to set the log level dynamically
+  on a per-request basis. To do so, set the `:log` option to
+  a tuple, `{Mod, Fun, Args}`. The `Plug.Conn.t()` for the
+  request will be prepended to the provided list of arguments.
+
+  When invoked, your function should return one of:
+
+    * a valid [`Logger.level()`](`t:Logger.level()/0`)
+    * `false` to disable logging
+    * `true` to use the event's default log level
```

Comment:
I would rather not support true for now and have users be explicit about it in their code. Thoughts?

Reply:
My thoughts are it's late and I thought you meant _we_ explicitly declare the default. :)

Reply:
Done :)